hawk_test_driver.py

```diff
@@ -195,12 +195,12 @@ def check_and_click_by_xpath(self, errmsg, xpath_exps):
                 print(f"ERROR: Couldn't find element by xpath [{xpath}] {errmsg}")
                 self.test_status = False
                 return
-            time.sleep(5)
+            time.sleep(1)
```

Comment:
this one come from https://github.com/ClusterLabs/hawk/pull/252/files#diff-a47168dfe2e7dd6f3e97f2169a7e168350b84ed87f6fc664cf77f4a1ca446f18R198